best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
+(e.g. `CreatePurchase`, `UpdateCreditCard`).
+* Expose one public method on services that does the work
```

Comment:
Some things it might be good to clarify:

* Should it be the *only* public method?
* Should it be a class method? Or an instance method?
* If it's an instance method, should the initializer accept arguments or state?